runtime/src/main/resources/reference.conf

```diff
@@ -54,9 +58,9 @@ akka.grpc.client."*" {
   # interpreted as retrying 'indefinitely'.
   connection-attempts = 20
 
-  # Service discovery mechamism to use. The default is to use a static host
-  # and port that will be resolved via DNS.
-  # Any of the mechanisms described in https://doc.akka.io/docs/akka-management/current/discovery/index.html can be used
-  # including Kubernetes, Consul, AWS API
+  # Request that the client try to connect the service immediately when the client is created
+  # rather than on the first request. Only supported for the Netty client backend, the akka-http client backend
+  # is always eager.
```

Comment:
```suggestion
  # Request that the client tries connecting to the service immediately when the client is created
  # rather than on the first request. Only supported for the Netty client backend, the Akka HTTP client backend
  # is always eager.
```